hooks/useSound.ts

```diff
@@ -0,0 +1,18 @@
+import { useEffect, useRef, useState } from "react";
+import { Sounds } from "./Sounds";
+
+export const useSound = () => {
+  const [isSoundLoaded, setIsSoundLoaded] = useState(false);
+  const soundRef = useRef(
+    new Sounds(
+      () => setIsSoundLoaded(true),
+      () => setIsSoundLoaded(false),
+    ),
+  );
+
+  useEffect(() => {
+    soundRef.current.stopAndUnload();
+  }, []);
```

Comment:
Is this supposed to be
```suggestion
useEffect(() => () => soundRef.current.stopAndUnload(), [])
```

Reply:
Nice catch 🎣  I changed this in [52cc381](https://github.com/equinor/attenuation-app/pull/28/commits/52cc381ab4d2981ced471e48fec9fb8af6a488d4).